README.md

```diff
@@ -6,14 +6,25 @@ This repository contains Helm charts used by Mina Foundation.
 
 The following is a list of charts available in this repository:
```

Comment:
**thought:** I think we should avoid to have to maintain a list of helm-charts in this main readme, given they are already listed as directories in the same repo. I think the list would not give much more values, we force us to maintain it, and most likely could become outdated. That being said, we could keep it and automate it too. Let's discuss about it in meeting another time.

Reply:
I think removing it is the best idea

Reply:
I will leave it though, and ask the DevOps channel on what it thinks.